python/langsmith/run_helpers.py

```diff
@@ -310,6 +311,8 @@ def traceable(
             Defaults to None.
         process_outputs: Custom serialization / processing function for outputs.
             Defaults to None.
+        dangerously_allow_filesystem: Whether to allow filesystem access for attachments.
+            Defaults to False.
```

Comment:
```suggestion
        dangerously_allow_filesystem: Whether to allow filesystem access for attachments.
            Defaults to False.
            
            Traces that reference local filepaths will be uploaded to LangSmith.
            In general, network-hosted applications should not be using this because
            referenced files are usually on the user's machine, not the host machine.
```

Some clarification like this probably good